### DISCUSSION TITLE
My Education Document is getting rejected even though it's valid

### DISCUSSION THREAD

**Discussion by namankumar80510** [community/community#58968] [GitHub Education]
### Hi everyone,

Hi. I am attempting to verify my college docs for GitHub education developer pack. I am attempting to submit the screenshot of my college portal that clearly verifies the validity of my college enrollment. But my application gets rejected instantly with this message:
> Unfortunately, we weren't able to approve your educational discount request for [@namankumar80510](https://github.com/namankumar80510).

 > What happened? The item you uploaded is insufficient to demonstrate your current academic status.
Please see suggestions. Suggested proof to expedite your approval:

> Your image contains keywords indicating that it may be an acceptance letter. Only proof of current academic enrollment is acceptable.

That is not the case in reality. Only an enrolled student can have access to this portal. Here is a screenshot:
![Screenshot_20230624_122518](https://github.com/community/community/assets/72571211/a2dec773-6304-4732-baa8-26d863e7f6e0)

**Comment by samus-aran** [comment]
Hi @namankumar80510 👋 

Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.

Hope this helps 👍
